taas/mapping.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Mapping(object):
+    """
+        This class handles mappings between our raw input, and what we generate.
+    """
+
+    # We're an abstract class
+    __metaclass__ = ABCMeta
+
+    # This was originally called 'value', but apparently that's a
```

Comment:
I love having (arbitrary) rules and conventions. I think the `NOTE` keyword is great for this context! Thanks, @Rtzq0 !